public/app/plugins/datasource/cloud-monitoring/components/SLO/SLOQueryEditor.tsx

```diff
@@ -29,6 +31,7 @@ export const defaultQuery: (dataSource: CloudMonitoringDatasource) => SLOQuery =
   serviceName: '',
   sloId: '',
   sloName: '',
+  lookbackPeriod: '1h',
```

Comment:
```suggestion
  lookbackPeriod: '',
```

Reply:
Since selector_slo_burn_rate requires the lookback period, INVALID_ARGUMENT error happens when first-switching the selector name from SLI Value (default) to SLO Burn rate as below: 
![image](https://user-images.githubusercontent.com/8341422/185508329-3609f468-73ee-4708-9d1b-bddae72706a0.png)
What do you think about that? I think it would be great if we can stop triggering the query if the lookback period is empty of undefined.

Reply:
you can use the method "filterQuery" of the datasource for that, see how is checking other parameters: 

https://github.com/grafana/grafana/blob/d4f382892d8871030333295074ffe3aaa28baf93/public/app/plugins/datasource/cloud-monitoring/datasource.ts#L226-L229

Reply:
Thanks, that makes sense. Fixed https://github.com/grafana/grafana/pull/53710/commits/f2ddd6642d775b230b792f0a5729e8cd45a17513. Also defined select_burn_slo_rate as const https://github.com/grafana/grafana/pull/53710/commits/e4c697b6df5221393896f804628532950548ae7b